packages/amplify-provider-awscloudformation/src/utility-functions.js

```diff
@@ -17,6 +17,7 @@ const { fileLogger } = require('./utils/aws-logger');
 
 const logger = fileLogger('utility-functions');
 const { getAccountId } = require('./amplify-sts');
+const { getAwsConfig } = require('./configuration-manager');
```

Comment:
this isn't version bump, is this intentional?

Reply:
This is a separately committed change to support the API category in release